guides/source/action_text_overview.md

```diff
@@ -8,149 +8,132 @@ rich text content.
 
 After reading this guide, you will know:
 
-* How to configure Action Text.
-* How to handle rich text content.
-* How to style rich text content and attachments.
+* What Action Text is, and how to install and configure it.
+* How to create, render, style and customize rich text content.
+* How to handle attachments.
 
 --------------------------------------------------------------------------------
 
 What is Action Text?
 --------------------
 
-Action Text brings rich text content and editing to Rails. It includes
-the [Trix editor](https://trix-editor.org) that handles everything from formatting
-to links to quotes to lists to embedded images and galleries.
-The rich text content generated by the Trix editor is saved in its own
-RichText model that can be associated with any existing Active Record model in the application.
-Any embedded images (or other attachments) are automatically stored using
-Active Storage and associated with the included RichText model.
+Action Text facilitates the handling and display of rich text content. Rich text content is text that includes formatting elements such as bold, italics, colors, and hyperlinks, providing a visually enhanced and structured presentation beyond plain text. It allows us to create rich text content, store it in a table, then attach it to any of our models.
 
-When it's time to render the content, Action Text processes it by sanitizing its
-content so that it's safe to embed directly into the page's HTML.
+INFO: **The Trix Editor**<br><br>
```

Comment:
I noticed that the paragraphs starting with "Most WYSIWYG editors are wrappers around HTML’s contenteditable..." are copy/pasted into this guide from https://trix-editor.org/. If we link that maybe we could remove the 'sidebar' altogether?

Reply:
I've implemented this suggestion and it improves the flow - thanks!

Reply:
@bhumi1102 that's actually a good point, we already link to the trix website, so maybe we can drop the whole INFO box altogether now, and let trix document it, since it's very specific (and kind of unnecessary information from the Rails perspective and development for the most part?)